validator/src/cli.rs

```diff
@@ -1700,6 +1691,13 @@ fn get_deprecated_arguments() -> Vec<Arg<'static, 'static>> {
             .hidden(hidden_unless_forced())
             .takes_value(false)
             .help("Disable manual compaction of the ledger database (this is ignored)."),
+        Arg::with_name("skip_poh_verify")
+            .long("skip-poh-verify")
+            .takes_value(false)
+            .help(
+                "Deprecated, please use --skip-verification.\n\
+                 Skip ledger verification at validator bootup."
+            ),
```

Comment:
```suggestion
            .hidden(hidden_unless_forced())
            ),
```
Same as the other deprecated args, can you please make this one hidden now

Reply:
I was not sure if I should hide them right a way.
But it does make sense.  Done.